lib/Mojo/IOLoop/ForkCall.pm

```diff
@@ -91,7 +91,7 @@ sub _run {
       waitpid $child, 0;
 
       # attempt to deserialize, emit error and return early
-      my $res = eval { $deserializer->($buffer) };
+      my $res = eval { $deserializer->($buffer) } if $buffer;
```

Comment:
That article also says:

> If you wish to send out the frozen scalar to another machine, use "nfreeze" instead to get a portable image.

If you are forking to another machine this is probably not the module for you :-P